README.md

```diff
@@ -1,2 +1,28 @@
 # glTF Assets Viewer
 The [glTF Asset Generator](https://github.com/bghgary/glTF-Asset-Generator) README files uses this tool to view assets.
+
+# How to add your engine
+The url parameters for your engine loader should understand these arguments, separated by ampersand (&):
```

Comment:
Since it's now template-based, this isn't correct anymore.

Reply:
You're right, I'll remove that section.